testing.rst

```diff
@@ -50,7 +50,8 @@ Symfony application.
 
     Code coverage can be generated with the ``--coverage-*`` options, see the
     help information that is shown when using ``--help`` for more information.
-
+    To avoid the error "No code coverage driver is available" when you are generating code coverage in html you should install php-debug 
```

Comment:
I have 2 issues with this change.

1) There are multiple extensions available which provide code coverage (xdebug, pcov, phpdbg) and this makes it sound only php-debug can be used.
2) I think, we should also point to the installation instructions to make this actionable, i.e. how to fix it rather than just saying what needs to be fixed.

Reply:
I improved the documentation, please check